tests/pyccel/test_pyccel.py

```diff
@@ -1002,6 +1002,6 @@ def test_json():
 #------------------------------------------------------------------------------
 def test_reserved_file_name():
     with pytest.raises(ValueError) as exc_info:
-        libname = str(random.choice(tuple(python_builtin_libs))) + ".py"
+        libname = str(random.choice(tuple(python_builtin_libs))) + ".py" # pylint: disable=no-member
```

Comment:
The error that you are getting is not a no-member error. Please check codacy for the correct keyword